src/components/MenuGroup/MenuGroup.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {BrowserRouter as Router} from 'react-router-dom';
+import {render, screen, fireEvent} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import MenuGroup from './index';
+
+describe('MenuGroup component', () => {
+  test('renders without crashing', async () => {
+    render(
+      <Router>
+        <MenuGroup title="Test" urlBase="Test" />
+      </Router>,
+    );
+
+    expect(screen.getByTestId('MenuGroup')).toBeTruthy();
+  });
+
+  test('renders with customTestId "Custom" without crashing', async () => {
+    render(
+      <Router>
+        <MenuGroup dataTestId="Custom" title="Test" urlBase="Test" />
+      </Router>,
+    );
+
+    expect(screen.getByTestId('Custom')).toBeTruthy();
+  });
+
+  test('renders with title', async () => {
+    render(
+      <Router>
+        <MenuGroup title="Test" urlBase="Test" />
+      </Router>,
+    );
+    expect(screen.getByText('Test')).toBeTruthy();
+  });
+
+  test('renders with child components', async () => {
+    render(
+      <Router>
+        <MenuGroup title="Test" urlBase="Test">
+          <div>Child 1</div>
+          <div>Child 2</div>
+        </MenuGroup>
```

Comment:
See if you can add a prop `children` and use that instead. You will have to update index.tsx too.

Reply:
Do we really need to type check children as React.node? 

Children is a default prop in React.FC

Reply:
I know, but this is the pattern we've been following. If multiple tests use children. You could just update it in one place.
```
const baseProps : MenuGroupProps = {
  children: <h1>Hello</h1>
}
```

Reply:
Ok no probs. I’ll do it when I get home tonight